Chessington.GameEngine/Pieces/Piece.cs

```diff
@@ -20,5 +20,63 @@ public void MoveTo(Board board, Square newSquare)
             var currentSquare = board.FindPiece(this);
             board.MovePiece(currentSquare, newSquare);
         }
+
+        protected IEnumerable<Square> GetAvailableLateralMoves(Board board)
+        {
+            var zeroes = Enumerable.Repeat(0, 15);
+            var minusSevenToSeven = Enumerable.Range(-7, 15);
+
+            var moves = zeroes.Zip(minusSevenToSeven, (x,y)=>(x,y));
```

Comment:
There should be whitespace after comma and around the arrow

Reply:
Yep, fixed in 6c5cbf8